packages/phrase/src/phrase-api.ts

```diff
@@ -22,15 +22,19 @@ function _callPhrase(path: string, options: Parameters<typeof fetch>[1] = {}) {
     },
   }).then(async (response) => {
     console.log(`${path}: ${response.status} - ${response.statusText}`);
+
+    const secondsUntilLimitReset = Math.ceil(
+      Number.parseFloat(response.headers.get('X-Rate-Limit-Reset') || '0') -
+        Date.now() / 1000,
+    );
     console.log(
       `Rate Limit: ${response.headers.get(
         'X-Rate-Limit-Remaining',
       )} of ${response.headers.get(
         'X-Rate-Limit-Limit',
-      )} remaining. (${response.headers.get(
-        'X-Rate-Limit-Reset',
-      )} seconds remaining})`,
+      )} remaining. (${secondsUntilLimitReset} seconds remaining})`,
```

Comment:
```suggestion
      )} remaining. (${secondsUntilLimitReset} seconds remaining)`,
```

Reply:
🤦 it was literally in the example screenshot to. Thanks